### ISSUE TITLE
Decreased alanine in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6034]
**Preferred term label:** Decreased alanine in urine

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of alanine in the urine is below the normal range., PMID: 5477682

**Parent term (use hpo.jax.org/app)** Aminoaciduria HP:0003355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Normally, amino acids are reabsorbed and are not excreted in the urine. Therefore, there is no "abnormally low" alanine in urine finding.
The pubmed article is actually refering to an increased level of alanine in urine, which is this:
https://hpo.jax.org/app/browse/term/HP:0020078

https://pubmed.ncbi.nlm.nih.gov/5477682/
